tutorials/advanced_features/Output Modulation and EOM Mode.ipynb

```diff
@@ -242,8 +242,8 @@
     "To overcome these limitations, a channel can be equipped with an EOM that allows the execution of pulses with a higher modulation bandwidth. For now, EOM mode operation is reserved for `Rydberg` channels and works under very specific conditions:\n",
     "\n",
     "   1. EOM mode must be explicitly enabled (`Sequence.enable_eom_mode()`) and disabled (`Sequence.disable_eom_mode()`).\n",
-    "   2. A buffering time is automatically added before the EOM mode is enabled and after it is disabled, as it needs to be isolated from regular channel operation.\n",
-    "   3. When enabling the EOM mode, one must choose the amplitude and detuning value that all square pulses will have. These values will also determine a set of options for the detuning during delays, out of which one is value chosen.\n",
+    "   2. A buffering time is automatically added before the EOM mode is enabled and after it is disabled, as it needs to be isolated from regular channel operation. During the starting buffer, the detuning already goes to the value it will assume in between pulses (_i.e._ during delays).\n",
+    "   3. When enabling the EOM mode, one must choose the amplitude and detuning value that all square pulses will have. These values will also determine a set of options for the detuning during delays, out of which one chosen.\n",
```

Comment:
Not sure the end of the sentence makes sense.

Reply:
Which one, the sentence that was added? Should it rather be:
```
 During the starting buffer, the detuning goes to the value it will assume between EOM pulses (_i.e._ during delays).
```

Reply:
Yes, more clear now, thanks.